### ISSUE TITLE
2.38 still not visible in PlayStore - does it take longer than 1-2 days?

### ISSUE THREAD

**Issue by gahabana** [MoleMan1024/audiowagon#114]
I have 2.3.7 installed. Checked yesterday (28th of March) afternoon (1700 CET) no updates to 2.3.8 were visible.  This morning of 29th still no luck.  
Does it just take longer or something went wrong with Google PlayStore?
TX!!!

**Comment by gahabana** [comment]
thank you as always @MoleMan1024 , will be patient. in the past it took day at most not sure what google is at. i did send them feedback specific to AudioWagon. 

not sure if you paid attention to 2nd part - for some reason AudioWagon is not showing up at all as app available for Android Automotive. Neither in the car (Volvo) nor in the URL i shared. Maybe that's Google's fault as well - am not sure - just want you to be aware of.

tx!

**Comment by gahabana** [comment]
@MoleMan1024 - can you please check please if status is still 'under review' (i.e. pure google fault) or something else wrong with update not being yet visible in playstore ? thank you !

**Comment by MoleMan1024** [comment]
@gahabana : yes it is still under review. This is nothing new, has happened before in the past. Be patient.

**Comment by gahabana** [comment]
> @gahabana : yes it is still under review. This is nothing new, has happened before in the past. Be patient.

it's finally on Play Store :) Thank you !
